mm2src/mm2_main/src/ordermatch_tests.rs

```diff
@@ -20,6 +20,8 @@ use std::collections::HashSet;
 use std::iter::{self, FromIterator};
 use std::sync::Mutex;
 
+const LEGACY_SWAP_V: u32 = 1;
```

Comment:
Same as [above](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2324#discussion_r1923211805)

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2324/commits/e37c74e9dcd27a3022734e3e1ff25fa8537fa69e

Reply:
It doesn't seem to be done on web UI:

![image](https://github.com/user-attachments/assets/edcc7f00-a189-4537-beb4-e6c110f97c37)

Reply:
I am not sure if we are on the same point. What I am suggesting is, you can use `legacy_swap_version` instead of creating another constant here.

Reply:
> I am not sure if we are on the same point. What I am suggesting is, you can use `legacy_swap_version` instead of creating another constant here.

Oh got it, got it, sorry I misread message. For some reason, I thought about GUI 🤦